### PR TITLE
Create custom templates

### PR DESCRIPTION
**Pull request by nathanbegbie**

### CODE REVIEW

**Comment by nathanbegbie** [comment]
Still delivers static content, but framework is in place for templates to be modified using Progressive Enhancement approach

File: README.rst [hunk 1]
```diff
@@ -1,5 +1,35 @@
-Installation of Unicore CMS
-===========================
+A4W Style Guide
+================================
+
+For a general overview of Unicore's WoW, see our `WoW Documentation <http://ways-of-working.readthedocs.org/en/latest/process.html#our-front-end-development-process>`_. Overall, we use an approach known as Progressive Enhancement.
```
Comment (SteveBarnett) [inline README.rst]: Might be worth mentioning that these are Praekelt's Ways of Working, and using the full words rather than the acronym?

File: README.rst [hunk 2]
```diff
@@ -1,5 +1,35 @@
...
+
+HTML
+----
+
+We use Jinja2_ for our templating language. We use an `atomic-design`_-like pattern, so each small section of html code is stored as atoms in ``atoms.html``, larger sections as molecules in ``molecules.html`` and so on, pulling it all together to create each page as a ``.jinja`` file. An example of how this modularised system is implemented can be found here_.
```
Comment (SteveBarnett) [inline README.rst]: It's better to use descriptive link text than "here." How about "in the template directory of the unicore-cms repository"?

File: README.rst [hunk 3]
```diff
@@ -9,7 +39,7 @@ Installation of Unicore CMS
     $ source ve/bin/activate
     (ve)$ pip install -e .
 
-Running Unicore CMS for local development
+Running Unicore CMS A4W for local development
 -----------------------------------------
 
 This is a Pyramid_ application, that uses Elasticsearch.
```
Comment (SteveBarnett) [inline README.rst]: stray comma ಠ_ಠ

File: README.rst [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+
+HTML
+----
+
+We use Jinja2_ for our templating language. We use an `atomic-design`_-like pattern, so each small section of html code is stored as atoms in ``atoms.html``, larger sections as molecules in ``molecules.html`` and so on, pulling it all together to create each page as a ``.jinja`` file. An example of how this modularised system is implemented can be found here_.
+
+CSS
+---
+
+We use `Sass`_ to create our CSS files. It will need to be run locally to compile the ``.scss`` files into the ``.css`` files. See below on how to install and run sass. We set the `output style`_ to ``:compressed``
```
Comment (SteveBarnett) [inline README.rst]: Maybe rephrase the "It will" sentence?

> For now, you need to commit the source `.scss` files, and the complied `.css` files.

File: README.rst [hunk 5]
```diff
@@ -1,5 +1,35 @@
...
+
+The main Sass file is ``/unicorecmsa4w/static/sass/style.scss``. This imports partials from the ``sass`` directory. We use a ``link rel="stylesheet" media="screen and (min-width: 20em)"`` to pull in (the compiled version of) ``/unicorecmsa4w/static/sass/enhanced.scss``, (which pulls in a few additional partials from the ``sass`` directory), and a Google web font.
+
+The sass-generated css files are placed and referenced in ``/unicorecmsa4w/static/css``.
+
+JavaScript
+----------
+
+We `cut the mustard`_, checking for support of ``querySelector``,
```
Comment (SteveBarnett) [inline README.rst:23]: Why is this important?
Something like this?

> This means that lower-end browsers get served a fast, HTML and CSS only, experience, while higher-end browsers get JavaScript enhancements.

File: README.rst [hunk 6]
```diff
@@ -60,6 +90,26 @@ The output of this should be roughly something like the following::
 Now loading http://localhost:8000/ should show the running site with
 the default content.
 
+The website will not have a css file yet yet. To generate the CSS files, you must first install sass using homebrew
```
Comment (SteveBarnett) [inline README.rst]: yet yet.

I think it might be nice to split this out a tiny bit? Let the sass docs handle talking about installing sass?

> You'll need to generate the CSS files from the Sass. You'll need to [install sass](http://sass-lang.com/install) if you haven't already.

File: README.rst [hunk 7]
```diff
@@ -60,6 +90,26 @@ The output of this should be roughly something like the following::
...
+.. code-block:: bash
+
+    $ sass unicorecmsa4w/static/sass/style.scss:unicorecmsa4w/static/css/style.css --style compressed
+    $ sass unicorecmsa4w/static/sass/enhanced.scss:unicorecmsa4w/static/css/enhanced.css --style compressed
+
+To run the pyramid application that will monitor changes in the sass files and recompile, allowing continuous changes, first ensure that you have followed the installation instructions above. Navigate to the repo root, make sure you have no other processes running in the background and then run:
+
+.. code-block:: bash
+
+    $ sh start.sh
```
Comment (SteveBarnett) [inline README.rst:104]: Maybe link to the `start.sh` file and say more clearly what it does (run pyramid server and sasquatch)?

File: start.sh [hunk 8]
```diff
@@ -0,0 +1,8 @@
+echo 'starting ve'
+virtualenv ve
+source ve/bin/activate
+echo 'Starting webserver..'
+pserve development.ini --reload &
+sass --watch unicorecmsa4w/static/sass:unicorecmsa4w/static/css --style compressed 
+kill %1
+echo 'done'
```
Comment (SteveBarnett) [inline start.sh:8]: Awesome that you got this working! \o/

File: README.rst (hunk 2 above)
Comment (SteveBarnett) [inline README.rst]: Might also be worth mentioning that we are using placeholder images? Ahem.

**Comment by SteveBarnett** [comment]
Apart from a few tiny comments, looks great! :+1:

**Comment by miltontony** [comment]
:+1: from me too once @SteveBarnett is satisfied

**Comment by nathanbegbie** [comment]
@SteveBarnett Thine will has been done. Probably.

**Comment by SteveBarnett** [comment]
:+1:
